MANIFEST.in

```diff
@@ -5,7 +5,6 @@ include img_src/oxygen_icon_set/*
 include MANIFEST.in
 include README.md
 include LICENSE.txt
-include CHANGELOG.md

```

Comment:
Should not this be the path to the changelog file related to Spyder 5? (And then should be changed when releasing Spyder 6 and newer major versions)

Reply:
It could if we'd like to keep including our changelog in our sdist and wheel. But I don't know if that's unnecessary.

I can change this to include our Spyder 5 changelog. What do you think?

Reply:
I think is better to include it :+1: